pkg/metrics/metrics.go

```diff
@@ -39,6 +39,10 @@ var (
 	// names and separated with a '_'
 	Namespace = "cilium"
 
+	// Datapath is the subsystem to scope metrics related to management of
+	// the datapath. It is prepended to metric names and separated with a '_'.
+	Datapath = "datapath_management"
```

Comment:
Namespace is a requirement, it will cause the `cilium_` to preface every metric. We should make sure to include that. Namespace is higher than subsystem.

For example, this shows up (with latest version) as:
```
cilium_datapath_errors_total{area="conntrack",family="ipv4",name="dump_reset"} 0
cilium_datapath_errors_total{area="conntrack",family="ipv6",name="dump_reset"} 0
```

In retrospect I might have liked the "drop" and so on to have their own subsystem, but it's not overly concerning. I think this is just how the prometheus library constructs the names of the metrics.

Reply:
yes I think I confused `subsystem` with `namespace` . But my initial concern was about the data path metrics which we already expose, which do not fall under this new subsystem this PR is introducing. Again this is more of a philosophical question to have more structured changes going forward as we incorporate more metrics.